notebooks/00_quick_start/naml_synthetic.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<p>Could you explain what "vert" and "subvert" mean here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1080/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
It means news vertical and subvertical.